gcp/cloud-endpoints/base/profile.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeflow.org/v1alpha1
+kind: Profile
+metadata:
+  name: kubeflow-admin
```

Comment:
Profile scope is `Cluster`, we'd better not hard-code its name here.

Reply:
namespace is injected since kustomization.yaml created after `kfctl generate all` includes a namespace

appdir/kustomize/cloud-endpoints/kustomization.yaml
```
apiVersion: kustomize.config.k8s.io/v1beta1
bases:
- base
kind: Kustomization
namespace: dls-kf
```

Reply:
this should be a function of email however email contains invalid characters 
for example `kam.d.kasravi@intel.com` both the '.' and the '@' are illegal.

We probably need to insert something which maps '.' => '-' and '@' => '-at-'